test/properties-test.js

```diff
@@ -151,6 +151,12 @@ describe('Properties', () => {
         compilerProps.get(languages, 'bar', true, undefined).should.deep.equal({a: false});
         compilerProps.propsByLangId[languages.a.id] = undefined;
     });
+    it('should not parse version properies as numbers', () => {
+        should.equal(casesProps('libs.example.versions.010.version'), '0.10');
```

Comment:
should.equal uses coercive equality comparison, so these tests will probably succeed even if the prop is parsed as a number since `0.1 == '0.10'` in JavaScript.

You should use `should.strictEqual` instead.

Reply:
`strictEqual` doesn't seem to be a method on `should`. Seems to be a difference between the [`assert`](https://www.chaijs.com/api/assert/#method_equal) (non-strict) and [`expect`/`should`](https://www.chaijs.com/api/bdd/#method_equal) (strict) APIs.

I've verified that the test fails without the fix.

Reply:
Woops - I was incorrectly looking at the should.js doc instead of the chai.js doc.